migrate-to-chimera

```diff
@@ -0,0 +1,144 @@
+#!/bin/bash
+
+arg=$1
+if [ $# -gt 1 ] ; then
+    echo "This script get at most one argument"
+    echo "Usage:"
+    echo "$0 [-l|--logout]"
+    echo ""
+    echo "  -l | --logout   Logout after migration"
+fi
+
+if ! [ -d $HOME/.config/steam-buddy ] && \
+    ! [ -d $HOME/.local/share/steam-buddy ] && \
+    ! [ -d $HOME/.cache/steam-buddy ] ;
+then
+    # Nothing to migrate, we can exit
+    echo "Found nothing to migrate"
+    exit 0
+fi
+
+## MIGRATION MAGIC ##
+
+# Go to our directories and replace settings:
+if [ -f $HOME/.local/share/steam-buddy/settings/settings.json ]; then
+    # Migrate settings to new location
+    echo "Moving configuration to new location"
+    if ! [ -d $HOME/.config/chimera ] ; then
+        mkdir -p $HOME/.config/chimera
+    fi
+    cp -bv $HOME/.local/share/steam-buddy/settings/settings.json $HOME/.config/chimera/settings.json
+    rm -rf $HOME/.local/share/chimera/settings
```

Comment:
is this assuming the top level directory has been moved already?
Perhaps just move the settings file directly from the original location?

Reply:
Tat rm line won't have any effect. It's probably a leftover from the old ordering. Removed